guide/start/policies.md

```diff
@@ -9,169 +9,423 @@ children:
+</pre>
+
+<!-- COMMENT:
+You can view its activity, to see the call to restart, using:
+
+{% highlight bash %}
+br application "Tomcat Cluster" entity "Cluster" entity "Tomcat Server" activity
 {% endhighlight %}
 
+TODO Why doesn't the restart() show in the activity view?!
+-->
+
+After the given `stabilizationDelay`, the Tomcat server will be automatically restarted.
+
+
+## Cluster auto-replace
+
+The cluster has a `ServiceReplacer` policy. This attaches to a DynamicCluster and replaces a 
+failed member; if this fails, it sets the Cluster state to on-fire.
 
-The loadbalancer's `Controller targets tracker` policy ensures that the loadbalancer is updated as the cluster size changes.
+To simulate a terminal the failure of a member, repeatedly kill the process (using the command 
+above).
 
-This policy automatically scales the cluster up or down to be the right size for the cluster's current load. One server 
-is the minimum size allowed by the policy.
+The Tomcat server should be replaced by a new member of the cluster, and then the old failed member
+removed.
 
-Sitting idle, this cluster will only contain one server, but you can use a tool like [jmeter](http://jmeter.apache.org/) 
-pointed at the nginx endpoint to create load on the cluster. Download a jmeter test 
-plan [here](https://github.com/apache/incubator-brooklyn/blob/master/examples/simple-web-cluster/resources/jmeter-test-plan.jmx).
+You can view the list of Tomcat servers in the cluster with the command below (which drills into the  
+application named "Tomcat Cluster", then into its child entity named "Cluster", and then lists the 
+child entities):
 
-As load is added, Apache Brooklyn requests a new cloud machine, creates a new app server, and adds it to the cluster. 
-As load is removed, servers are removed from the cluster, and the infrastructure is handed back to the cloud.
+{% highlight bash %}
+br application "Tomcat Cluster" entity "Cluster" entity
+{% endhighlight %}
+
+<pre>
+ Id         Name            Type   
+ dYfUvLIw   quarantine      org.apache.brooklyn.entity.group.QuarantineGroup   
+ tOpMeYYr   Tomcat Server   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+ mgoRpkKH   Tomcat Server   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+</pre>
 
 
-### Under the Covers
+## Auto-scaling
 
-The `AutoScalerPolicy` here is configured to respond to the sensor
-reporting requests per second per node, invoking the default ```resize``` effector.
-By updating on the policy, you can configure it to respond to a much lower threshhold
-or set long stabilization delays (the period before it scales out or back).
+The `AutoScalerPolicy` attaches to a `Resizable` entity (in this case the `DynamicCluster`) and 
+dynamically adjusts its size in response to either keeping a metric within a given range, or in 
+response to Pool Cold and Pool Hot events. It adds/removes members (e.g. Tomcat instances) 
```

Comment:
I'll take it out of the getting started guide - a separate how-to (not yet written!) about the auto-scaling policy could talk about more advanced use-cases.

FYI the hot/cold events are a way that an external system (or external enricher) can trigger auto-scaling. Instead of having some metric kept in a given range, something else can figure out what needs to be done and then emit these events to tell the auto-scaler that the cluster is too hot (i.e. over-utilised, so too small to handle the current workload) or too cold (i.e. under-utilised). This is only used by a small number of (old) entities.